src/assets/i18n/it.po

```diff
@@ -4203,35 +4218,54 @@ msgid ""
 "not accept Minute values. After the time  expires, the snapshot is removed. "
 "Snapshots which have been replicated  to other systems are not affected."
 msgstr ""
+"Definire un periodo di tempo per la conservazione dello snapshot su questo "
+"sistema utilizzando un valore numerico e una singola lettera minuscola per le "
+"unità. Esempi: <i>3h</i> per 3 Ore, <i>1m</i> per 1 Mese, and <i>1y</i> per 1"
```

Comment:
Esempio:

Reply:
eh questa è problematica: ho tradotto "per le unità" letterelamente ma non ha senso in ita.

ci penso su.